### ISSUE TITLE
Post request loosing content and not getting correct response

### ISSUE THREAD

**Issue by r7l** [clue/php-buzz-react#98]
Please excuse if this is not the right place to ask but i haven't found any forum or chat.

I am trying to do a POST request with Buzz to another server that is running some Slim PHP application. Somehow the content is lost on the receiving end and the same in reverse. The request itself comes through perfectly and the other code works as well as it was tested by using some JS.

The other thing would be the response. It is always giving me **null** no matter what i am sending back.

Is there anything i am missing? The request itself is correctly identified as POST on the recieving end and it works fine with all my GET requests. The documentation or examples are slim on that topic.

Here is the testing code that is not working for me (i need those TLS options because of self signed SSL):

```
$content['var'] = 'bla';

$content = http_build_query($content);

$connection = new Connector($loop, [
    'tls' => [
        'verify_peer' => false,
        'verify_peer_name' => false,
        'allow_self_signed' => true,
   ]
]);

$browser = new Browser($loop, $connection);

$browser = $browser->withOptions(array(
    'followRedirects' => true,
    'maxRedirects' => 10,
    'obeySuccessCode' => true,
    'streaming' => false,
));

$browser->post($url, array(), $content)->then(function(ResponseInterface $response) {
    $output = (string)$response->getBody();
});```

**Comment by r7l** [comment]
Thanks allot, it actually helped. Would have expected it be filled with needed header already. It might be a good idea to add this into the list of methods in the Readme file. That's why i was failing.

**Comment by clue** [comment]
I agree that the documentation could absolutely use a remark to highlight this very issue, do you feel like filing a simple documentation PR? :shipit: :+1: 

Unfortunately we can't really apply this as a sane default here as different HTTP endpoints use different encodings. But this is certainly something that still applies to a large number of use cases, so a word of warning makes perfect sense :+1:
